docs/summary.md

```diff
@@ -11,16 +11,16 @@
 
 ## Deployments
 
-* [Sync API](deployments/syncapi.md)
-  * [Predictor implementation](deployments/syncapi/predictors.md)
-  * [API configuration](deployments/syncapi/api-configuration.md)
-  * [API deployment](deployments/syncapi/deployment.md)
-  * [API statuses](deployments/syncapi/statuses.md)
-  * [Parallelism](deployments/syncapi/parallelism.md)
-  * [Autoscaling](deployments/syncapi/autoscaling.md)
-  * [Prediction monitoring](deployments/syncapi/prediction-monitoring.md)
+* [Realtime API](deployments/realtime.md)
+  * [Predictor implementation](deployments/realtime/predictors.md)
+  * [API configuration](deployments/realtime/api-configuration.md)
+  * [API deployment](deployments/realtime/deployment.md)
+  * [API statuses](deployments/realtime/statuses.md)
+  * [Parallelism](deployments/realtime/parallelism.md)
+  * [Autoscaling](deployments/realtime/autoscaling.md)
```

Comment:
Is it supposed to be `deployments/realtime/` or `deployments/realtimeapi/`? I think both have been used, and I think I actually prefer `deployments/realtime-api/`; we can discuss tomorrow